.github/workflows/main.yaml

```diff
@@ -1,6 +1,8 @@
 name: Release Containerimage
 
 on:
+  schedule:
+    - cron: "0 7 * * 1"
```

Comment:
Den Schedule kannst du hier wieder entfernen. In diesem Projekt brauchen wir das nicht.